compiler/rustc_error_messages/locales/en-US/borrowck.ftl

```diff
@@ -123,3 +162,90 @@ borrowck_cannot_move_when_borrowed =
         [value] value
         *[other] {$value_place}
     } occurs here
+
+borrowck_moved_due_to_call =
+    {$place_name} {$has_partially_str ->
+        [true] partially {""}
```

Comment:
```suggestion
        [true] partially
```
similarly elsewhere?

Reply:
Since it's not the first word in the sentence, I think I could really use a space..

Reply:
oh I get it now, I can have a trailing space, I thought tidy disapproved it once..

Reply:
yeah `tidy` won't allow it:
```
tidy error: \borrowck.ftl:125: trailing whitespace
some tidy checks failed
```

Reply:
`./x.py test tidy --bless` won't work.